### ISSUE TITLE
Deprecation warning wen used with Angular 5

### ISSUE THREAD

**Issue by vajahath** [PolymerElements/paper-spinner#75]
### Description
Chrome poses a deprecation warning for html imports.
```
[Deprecation] Styling master document from stylesheets defined in HTML Imports is deprecated, and is planned to be removed in M65, around March 2018. Please refer to https://goo.gl/EGXzpw for possible migration paths.
```

### Steps to reproduce
create a new Angular app with cli.
main `index.html` looks like:
```html
<!doctype html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <title>Mig</title>
  <base href="/">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="icon" type="image/x-icon" href="favicon.ico">

</head>

<script src="/assets/bower_components/webcomponentsjs/webcomponents-lite.js"></script>
<link rel="import" href="/assets/bower_components/paper-spinner/paper-spinner-lite.html">

<body>
  <app-root></app-root>
</body>

</html>
```
my `viewer.component.html` (where I use the element) looks like:
```html
<!-- <style is="custom-style"> </style> goes here -->
<h3>Viewer</h3>
<paper-spinner-lite active class="orange"></paper-spinner-lite>
```

### Browsers Affected
just checked in chrome. not sure about others.
(Version 63.0.3239.108 (Official Build) (64-bit)

**Comment by keanulee** [comment]
TL;DR Polymer will make sure that styles still work, but Polymer can't remove that console warning. More info: https://www.polymer-project.org/blog/2017-10-18-upcoming-changes.html (under heading "HTML imports styling the main document")
